src/main/java/games/strategy/triplea/ai/proAI/data/ProPurchaseTerritory.java

```diff
@@ -18,15 +18,22 @@
 
   public ProPurchaseTerritory(final Territory territory, final GameData data, final PlayerID player,
       final int unitProduction) {
+    this(territory, data, player, unitProduction, false);
+  }
+
+  public ProPurchaseTerritory(final Territory territory, final GameData data, final PlayerID player,
+      final int unitProduction, final boolean isBid) {
```

Comment:
"Missing a Javadoc comment."

I would say at least documenting the new parameter (`isBid`) would be helpful and get rid of the warning at the same time.  For someone like me who knows nothing about the AI code, little things like explaining what the consequences of bidding or not bidding are might make it easier to learn. :smiley: